pkg/rpc/server_test.go

```diff
@@ -298,6 +298,41 @@ var rpcTestCases = map[string][]rpcTestCase{
 			fail:   true,
 		},
 	},
+	"getblocksysfee": {
+		{
+			name:   "positive",
+			params: "[1]",
+			result: func(e *executor) interface{} { return "" },
+			check: func(t *testing.T, e *executor, result interface{}) {
+				res, ok := result.(*StringResultResponse)
+				require.True(t, ok)
+
+				block, err := e.chain.GetBlock(e.chain.GetHeaderHash(1))
+				require.NoErrorf(t, err, "could not get block")
+
+				var expectedBlockSysFee util.Fixed8 = 0
```

Comment:
It is initialized to `0` by default, there is no need to specify it explicitly.

Reply:
Fixed